.env.example

```diff
@@ -1,25 +1,26 @@
 # Example .env file for selfhosting the bot
 # Don't add your .env file to your github repository, these settings are used to access the bot
 
-
+# To edit/set the settings, remove this line and the one below it
+READONLY=true
```

Comment:
Read-only mode is enabled through `npm run readonly` to avoid the user having to edit the `.env` file to use it.

Also the example file shouldn't be in read-only mode as most won't use that.